sagemaker-training-compiler/huggingface/pytorch_single_gpu_single_node/albert-base-v2/albert-base-v2.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>No need to do code formatting for dataset names or class names - code formatting is for code only</p><p> </p><p>But you can keep code formatting for code like <code>datasets.load_dataset()</code> or  </p><pre class="ql-syntax" spellcheck="false">from datasets import load_dataset
load_dataset('dataset_name')
</pre><p>as they are code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I used code format because this "SQuAD" and <code>"HuggingFace DatasetDict"</code>didn't pass grammar check. This is the grammar log at that time:
</p><pre class="ql-syntax" spellcheck="false">Offset 43, length 5, Rule ID: MORFOLOGIK_RULE_EN_US
Message: Possible spelling mistake found.
Suggestion: Squad; Squads; Quad; Squid; Squat; Squaw; Squab; Square; Sound; Equal; Sued; Sad; Squash; Quid; Salad; Squared; Aqua; Quay; Stead; Quads; Shad; Squalid; Squall; Squats; Scud; Snead; Skuas; Squaws; Squeak; Squeal; Squids; Scad; Skua; Spud; Squawk; Squib; Squabs; Aquas; SAD; SIAD; SOAD; SOUD; SPAD; SUA; SUD; UAD; Qua; Slued; Stud; Shead
... this example notebook, we prepared the SQuAD v1.1 dataset in the public SageMaker sa...

Offset 206, length 11, Rule ID: MORFOLOGIK_RULE_EN_US
Message: Possible spelling mistake found.
...he dataset and convert to a HuggingFace DatasetDict.
</pre><p> </p>